exploratory_analysis.ipynb

```diff
@@ -1825,136 +1758,136 @@
        "    <tr>\n",
-       "      <th>247718</th>\n",
-       "      <td>KY</td>\n",
-       "      <td>McLean County</td>\n",
-       "      <td>21.0</td>\n",
-       "      <td>149.0</td>\n",
-       "      <td>Violent crime rate</td>\n",
-       "      <td>43.0</td>\n",
-       "      <td>2.666667</td>\n",
-       "      <td>9597.333333</td>\n",
-       "      <td>27.785496</td>\n",
-       "      <td>21149.0</td>\n",
-       "      <td>2009.0</td>\n",
+       "      <th>193</th>\n",
+       "      <td>Diabetic screening</td>\n",
+       "      <td>TX</td>\n",
+       "      <td>741</td>\n",
+       "      <td>741</td>\n",
+       "      <td>741</td>\n",
+       "      <td>741</td>\n",
+       "      <td>0</td>\n",
+       "      <td>741</td>\n",
+       "      <td>741</td>\n",
+       "      <td>741</td>\n",
+       "      <td>741</td>\n",
        "    </tr>\n",
        "  </tbody>\n",
        "</table>\n",
        "</div>"
       ],
       "text/plain": [
-       "       state           county  state_code  county_code        measure_name  \\\n",
-       "245430    IN     Boone County        18.0         11.0  Violent crime rate   \n",
-       "247672    KY     Lewis County        21.0        135.0  Violent crime rate   \n",
-       "247732    KY  Magoffin County        21.0        153.0  Violent crime rate   \n",
-       "247725    KY   Madison County        21.0        151.0  Violent crime rate   \n",
-       "247718    KY    McLean County        21.0        149.0  Violent crime rate   \n",
-       "\n",
-       "        measure_id   numerator   denominator   raw_value  fips_code  \\\n",
-       "245430        43.0    9.500000  40952.000000   23.197890    18011.0   \n",
-       "247672        43.0    9.000000  13877.000000   64.855516    21135.0   \n",
-       "247732        43.0    4.666667  13304.000000   35.077170    21153.0   \n",
-       "247725        43.0  168.666667  83345.666670  202.370049    21151.0   \n",
-       "247718        43.0    2.666667   9597.333333   27.785496    21149.0   \n",
+       "                        measure_name state  county  state_code  county_code  \\\n",
+       "94               Children in poverty    TX     762         762          762   \n",
+       "142    Daily fine particulate matter    TX     762         762          762   \n",
+       "448                     Unemployment    TX     762         762          762   \n",
+       "397  Sexually transmitted infections    TX     742         742          742   \n",
+       "193               Diabetic screening    TX     741         741          741   \n",
        "\n",
-       "        start_year  \n",
-       "245430      2009.0  \n",
-       "247672      2009.0  \n",
-       "247732      2009.0  \n",
-       "247725      2009.0  \n",
-       "247718      2009.0  "
+       "     measure_id  numerator  denominator  raw_value  fips_code  start_year  \n",
+       "94          762        762            0        762        762         762  \n",
```

Comment:
This analysis nicely highlights what I was saying about how the data is stored differently for different measures. You can see some measures only include a raw_value, while other measures have just raw_value and numerator, etc. We should stick to using only raw_values in our analysis but this is useful context